jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -358,6 +365,32 @@ public Builder setRetries(Integer retries) {
         }
 
         /**
+         * Disable the "mea-culpa" retries.
+         *
+         * @see <a href ="https://github.com/twosigma/Cook/blob/master/scheduler/docs/faq.md#how-can-i-configure-my-job-to-run-exactly-once">
+         *      how-can-i-configure-my-job-to-run-exactly-once
+         *      </a>
+         * @return this builder.
+         */
+        public Builder disableMeaCulpaRetries() {
```

Comment:
It is more clear. Btw, this is first config for boolean. So prefer to use disable/enableXXX